src/execution/operator/aggregate/physical_streaming_window.cpp

```diff
@@ -179,6 +292,113 @@ unique_ptr<OperatorState> PhysicalStreamingWindow::GetOperatorState(ExecutionCon
 	return make_uniq<StreamingWindowState>(context.client);
 }
 
+void StreamingWindowState::AggregateState::Execute(ExecutionContext &context, DataChunk &input, Vector &result) {
+	//	Establish the aggregation environment
+	const idx_t count = input.size();
+	auto &aggregate = *wexpr.aggregate;
+	auto &aggr_state = *this;
+	auto &statev = aggr_state.statev;
+
+	// Compute the FILTER mask (if any)
+	ValidityMask filter_mask;
+	auto filtered = count;
+	auto &filter_sel = aggr_state.filter_sel;
+	if (wexpr.filter_expr) {
+		ExpressionExecutor filter_executor(context.client, *wexpr.filter_expr);
+		filtered = filter_executor.SelectExpression(input, filter_sel);
+		if (filtered < count) {
+			filter_mask.Initialize(count);
+			filter_mask.SetAllInvalid(count);
+			for (idx_t f = 0; f < filtered; ++f) {
+				filter_mask.SetValid(filter_sel.get_index(f));
+			}
+		}
+	}
+
+	// Check for COUNT(*)
+	if (wexpr.children.empty()) {
+		D_ASSERT(GetTypeIdSize(result.GetType().InternalType()) == sizeof(int64_t));
+		auto data = FlatVector::GetData<int64_t>(result);
+		auto &unfiltered = aggr_state.unfiltered;
+		for (idx_t i = 0; i < count; ++i) {
+			unfiltered += int64_t(filter_mask.RowIsValid(i));
+			data[i] = unfiltered;
+		}
+		return;
+	}
+
+	// Compute the arguments
+	ExpressionExecutor executor(context.client);
```

Comment:
Unrelated to this PR (as this is just moving the code) - it would be better to keep these cached as well if possible